app/views/manuals/_content_for_head_tag.html.erb

```diff
@@ -2,6 +2,7 @@
   <% if description.present? %>
     <meta name='description' content='<%= description %>' />
   <% end %>
+  <%= ab_test.requested_variant.analytics_meta_tag.html_safe if add_ab_meta_tag %>
```

Comment:
Instead of using this `if` could we not use a variant? That would be more consistent with using variants elsewhere, and would remove the need for passing around the `add_ab_meta_tag` flag.